tools/sanity_checks.py

```diff
@@ -105,6 +105,9 @@
     'leptonica': [
         '_skipped_test.c',
     ],
+    'libuv': [
+        'copy_file_in_build_dir.py',
+    ],
```

Comment:
This file is not actually used in the current build setup at least so this should not be here.

In any case it should use [configure_file instead](https://mesonbuild.com/Reference-manual_functions.html#configure_file).

Reply:
It's not used because now I'm trying custom_target() instead of run_command() as before, but I don't get it working anyway because custom_target() doesn't accept as input an autogenerated file (the test executable itself).

Reply:
NOTE: Finally I made `run_command()` work so the new `link_file_in_build_dir.py` script is finally needed here.